tests/integration/cli/action/test_action_classification.py

```diff
@@ -15,13 +15,11 @@
 from tests.test_suite.e2e_test_system import e2e_pytest_component
 
 # Finetuning arguments
+# TODO: Need to change sample dataset
 args = {
-    "--train-ann-file": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
-    "--train-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
-    "--val-ann-file": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
-    "--val-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
-    "--test-ann-files": "data/custom_action_recognition/custom_dataset/train_list_rawframes.txt",
-    "--test-data-roots": "data/custom_action_recognition/custom_dataset/rawframes",
+    "--train-data-roots": "/local/sungmanc/datasets/jester_SC_cvat_multifolder_classification/train",
```

Comment:
Please, don't forget to del the local paths later

Reply:
Sure, thanks